addon/components/au-date-picker.js

```diff
@@ -1,7 +1,9 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { action } from "@ember/object";
 import { assert, runInDebug } from '@ember/debug';
 import { guidFor } from '@ember/object/internals';
+import formatISO from 'date-fns/formatISO'
```

Comment:
FYI I verified that this was properly tree shaken and not all the date-fns were included:

Before this PR:
` - dist/assets/vendor-970d88007936a1cefb44c371d9bd4ca3.js: 2.28 MB (487.25 KB gzipped)`

Now:
` - dist/assets/vendor-5dd17018d26aba4293fbfe3e213a05bf.js: 2.28 MB (488.24 KB gzipped)`

So that seems about right.